dspace/modules/api/src/main/java/org/datadryad/authority/JournalConceptIndexer.java

```diff
@@ -17,31 +15,37 @@
  * Time: 2:42:07 PM
  * To change this template use File | Settings | File Templates.
  */
-public class FileIndexer implements AuthorityIndexerInterface {
+public class JournalConceptIndexer implements AuthorityIndexerInterface {
+
+    private String SOURCE="JOURNALCONCEPTS";
     private AuthorityValue nextValue;
 
-    LinkedList<AuthorityValue> authorities = new LinkedList<AuthorityValue>();
+    LinkedList<AuthorityValue> authorities = new LinkedList();
 
-    Context context;
+
+    public static final String FIELD_NAME = "prism_publicationName";
 
     public void init() {
-        DryadJournalConcept[] journalConcepts = JournalUtils.getAllJournalConcepts();
-        for (DryadJournalConcept journalConcept : journalConcepts) {
-            AuthorityValue authorityValue = createAuthorityValue(journalConcept);
-            authorities.push(authorityValue);
+        DryadJournalConcept[] dryadJournalConcepts = JournalUtils.getAllJournalConcepts();
+        for (DryadJournalConcept concept : dryadJournalConcepts) {
+            System.out.println("concept is " + concept.getFullName());
```

Comment:
Well, it probably shouldn't be there at all. It was System.out.println because I mostly saw it when running dspace from the command line, but I can delete it now.